strs.go

```diff
@@ -57,6 +57,41 @@ func (db *RoseDB) MGet(keys [][]byte) ([][]byte, error) {
 	return values, nil
 }
 
+// GetRange returns the substring of the string value stored at key,
+// determined by the offsets start and end.
+func (db *RoseDB) GetRange(key []byte, start, end int) ([]byte, error) {
+	db.strIndex.mu.RLock()
+	defer db.strIndex.mu.RUnlock()
+
+	val, err := db.getVal(db.strIndex.idxTree, key, String)
```

Comment:
if val is nil or `len(val) == 0`, return directly

Reply:
Yeah, that's right.